provider_code_spec.json

```diff
@@ -437,12 +438,9 @@
 										"imports": [
 											{
 												"path": "github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
-											},
-											{
-												"path": "regexp"
 											}
 										],
-										"schema_definition": "stringvalidator.RegexMatches(regexp.MustCompile(`^gen[12]$`), \"deployment_type must be gen1 (or) gen2\")"
+										"schema_definition": "stringvalidator.OneOf(\"gen1\", \"gen2\")"
```

Comment:
This should only contain `gen2`?

Reply:
Existing Gen1 cloudspaces will cease functioning (updates will fail) if validations are enforced. Therefore, both are permitted. Validations are applied in `ModifyPlan()`, where a distinction between Create and Update operations can be made.

Reply:
I see the reasoning behind restricting Gen1 through attribute validations, as users can still manage Gen1 cloudspaces with an older provider version.